packages/ceramic-doctype-account-link/src/account-link-doctype.ts

```diff
@@ -10,6 +10,10 @@ export interface AccountLinkParams extends DocParams {
     content?: object;
 }
 
+function isValidChainId(chainId: string) {
+    return chainId === 'eip155:1' || chainId === 'fil:t' || chainId === 'fil:m'
+}
```

Comment:
We should really support all caip-2 chainIDs that are supported by `3id-blockchain-utils`. So I don't think we need to check if it's valid here. We can just accept all of them!

Reply:
Yay! Makes total sense.